### PR TITLE
deprecate(addon-table): `*tuiRow` deprecation (use `*ngFor` + `tuiTableSort`-pipe)

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [X] Documentation content changes

## What is the current behavior?
```console
NG0303: Can't bind to 'tuiCellOf' since it isn't a known property of 'td'
(used in the 'AppComponent' component template).
1. If 'td' is an Angular component and it has the 'tuiCellOf' input, then verify that it is a part of an @NgModule where this component is declared.
2. To allow any property add 'NO_ERRORS_SCHEMA' to the '@NgModule.schemas' of this component.
```

## What is the new behavior?
No error

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo/src/modules/tables/table/examples/import/insert-template.md [hunk 1]
```diff
@@ -32,41 +32,44 @@
       </th>
     </tr>
   </thead>
-  <tbody tuiTbody>
+  <tbody
+    tuiTbody
+    [data]="users"
+  >
     <tr
-      *ngFor="let item of users | tuiTableSort"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/tables/table/examples/import/insert-template.md:37]: revert this

**Review by vladimirpotekhin — APPROVED** [review]

**Review by splincode — APPROVED** [review]

File: projects/addon-table/components/table/tbody/tbody.template.html [hunk 2]
```diff
@@ -22,6 +21,7 @@
         </button>
     </th>
 </tr>
+<ng-content></ng-content>
```
Comment (waterplea) [inline projects/addon-table/components/table/tbody/tbody.template.html:24]: This way you cannot close rows by clicking heading. I guess you need to add `*ngIf="open"`

File: projects/demo/src/modules/tables/table/examples/3/index.html [hunk 3]
```diff
@@ -74,28 +74,24 @@
...
-                    <tui-text-area
-                        class="textarea"
-                        [expandable]="true"
-                        [ngModel]="item.name"
-                        [ngModelOptions]="options"
-                        (ngModelChange)="onValueChange($event, 'name', item, pythons)"
-                    ></tui-text-area>
-                </th>
-                <ng-container *tuiCell="'price'">
+            <ng-container *ngIf="pythonSection.open">
```
Comment (waterplea) [inline projects/demo/src/modules/tables/table/examples/3/index.html:81]: Instead let's handle this internally.

File: projects/addon-table/components/table/directives/cell.directive.ts [hunk 4]
```diff
@@ -1,6 +1,12 @@
 import {Directive, Inject, Input, TemplateRef} from '@angular/core';
 import {tuiDefaultProp} from '@taiga-ui/cdk';
 
+/**
+ * @deprecated
+ * It is required only by {@link TuiTrComponent}. Can be dropped after its deletion.
+ * ___
+ * TODO v4.0 delete it.
+ */
 @Directive({
```
Comment (waterplea) [inline projects/addon-table/components/table/directives/cell.directive.ts:10]: tuiCell is necessary in order to rearrange columns by `columns` array of `tuiTable` directive.

**Review by waterplea — CHANGES_REQUESTED** [review]

File: projects/addon-table/components/table/directives/cell.directive.ts (hunk 4 above)
Comment (nsbarsukov) [reply projects/addon-table/components/table/directives/cell.directive.ts:10]: Screenshot 2022-10-12 at 14 39 17

File: projects/demo/src/modules/tables/table/examples/3/index.html (hunk 3 above)
Comment (nsbarsukov) [reply projects/demo/src/modules/tables/table/examples/3/index.html]: fixed

File: projects/addon-table/components/table/tbody/tbody.template.html (hunk 2 above)
Comment (nsbarsukov) [reply projects/addon-table/components/table/tbody/tbody.template.html]: fixed

File: projects/addon-table/components/table/directives/cell.directive.ts (hunk 4 above)
Comment (nsbarsukov) [reply projects/addon-table/components/table/directives/cell.directive.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Comment by nsbarsukov** [comment]
@splincode @vladimirpotekhin could you review again, please ?

**Pull request merged**